openstack/cross-layers/packetfunctions.c

```diff
@@ -303,6 +303,39 @@ void packetfunctions_tossFooter(OpenQueueEntry_t* pkt, uint8_t header_length) {
    }
 }
 
+//======= packet duplication
+
+void packetfunctions_duplicatePacket(OpenQueueEntry_t* dst, OpenQueueEntry_t* src) {

```

Comment:
please provide a description of what this function does, and how/when it if used.